spec/amber/amber_spec.cr

```diff
@@ -56,9 +56,9 @@ describe Amber do
         expected_session = {:key => "amber.session", :store => :signed_cookie, :expires => 0}
         settings.session.should eq expected_session
         settings.port.should_not eq 3000
+        settings.database.should eq "mysql://root@localhost:3306/amber_test_app_test"
```

Comment:
I liked this being in secrets more because as a framework we're not opinionated on which ORM to use or even if you have to use one at all. 

Someone might choose to use mongo instead of a relational database or choose to have 2 or 3 databases per environment. As such I feel like it makes more sense to leave this in secrets. 

WDYGT? @drujensen @marksiemers @faustinoaq @eliasjpr

Reply:
One way or another, the database url for non-development environments needs to be protected, as it has a username and password in it.

Either encrypting it, or reading from an environment variable.

Is there a certain one that amber has been preferring to date?